smithy-model/src/main/java/software/amazon/smithy/model/shapes/ResourceShape.java

```diff
@@ -87,6 +96,19 @@ public Builder toBuilder() {
         return Collections.unmodifiableSet(allOperations);
     }
 
+    /**
+     * Gets the operations bound through the "collectionOperations" property.
+     *
+     * This will not include operations bound to resources using a lifecycle
+     * operation binding.
+     *
+     * @return Get the "collectionOperations" directly bound to this shape.
+     * @see #getAllOperations()
+     */
+    public Set<ShapeId> getCollectionOperations() {
```

Comment:
Should this now also have a `getInstanceOperations` method?

Reply:
This is specifically getting what's bound to `collectionOperations`, the equivalent would be `getOperations`